README.md

```diff
@@ -1,70 +1,121 @@
+## Authors
 
-### Code Splitting
+👤 **Petro Loltolo Lesapiti**
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+- GitHub: [@petrolesapiti](https://github.com/Loltolo-Lesapiti)
+- LinkedIn: [@petrolesapiti](https://www.linkedin.com/in/petrolesapitiloltolo/)
 
-### Analyzing the Bundle Size
+## 🤝 Contributing
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+Contributions, issues, and feature requests are welcome!
 
-### Making a Progressive Web App
+## Getting Started
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+To get a local copy follow these simple steps:
 
-### Advanced Configuration
+**Clone Your Github Repository**
+Follow these steps to clone your
+**Clone Your Github Repository**
+Follow these steps to clone my Github repo on Windows:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+1. Open Git Bash;
 
-### Deployment
+- If Git is not already installed, it is super simple. Just go to the Git Download Folder and follow the instructions.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+2. Go to the current directory where you want the cloned directory to be added;
 
-### `npm run build` fails to minify
+- To do this, input cd and add your folder location. You can add the folder location by dragging the folder to Git bash.
+- $ cd '/c/Users/Documents/My Git Project'
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+3. Go to the page of the repository that you want to clone.
+
+4. Click on “Clone or download” and copy the URL.
+5. Use the git clone command along with the copied URL from earlier;
+
+- $ git clone https://
+
+6. Press Enter;
+
+- $ git clone https://
+  Cloning into Git …
+  remote: Counting objects: 13, done.
+  remote: Compressing objects: 100% (13/13), done.
+  remove: Total 13 (delta 1), reused 0 (delta 1)
+  Unpacking objects: 100% (13/13), done.
+  **Congratulations, you have created your first local clone from your remote Github repository.**
+
+  ## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://.
+
+## Show your support
+
+Give a ⭐ if you like this project!
+
+## Acknowledgments
+
+## Acknowledgments
+
+- Thanks to Nelson Sakwa for original design.
```

Comment:
- [ ] [OPTIONAL] Great job acknowledging `Nelson Sakwa`. 👏  It will be great if you also add a link to his [work](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) so it will be accessible to anyone interested 🙏